Corpus/residential.py

```diff
@@ -364,11 +364,26 @@ def receptacles(self):
                 eq = Equipment(**set_appliances[app])
                 r_app = dict()
                 n_app = 0
-                # loop for all household mmembers
+                # loop for all household members
                 for i in counter:
-                    r_appi, n_appi = eq.simulate(nday, dow, self.clusters[i], self.occ[i])
-                    r_app = stats.sum_dict(r_app, r_appi)
-                    n_app += n_appi
+                    # if appliances are not shared the load of using the appliance can be summed for each householdmember
+                    if app in ('CordlessPhone','PC'): # it is assumed that only the cordlessphone and PC are not shared. Each person in the household (>12) has its own phone or PC.
+                        r_appi, n_appi = eq.simulate(nday, dow, self.clusters[i], self.occ[i])
+                        r_app = stats.sum_dict(r_app, r_appi)
+                        n_app += n_appi
+                    else:
+                        # if appliances are shared the load of using the appliance can be maximal the cycle load. 
```

Comment:
I noticed the run time increased a lot. A first suggestion would be to move the "continuous loads" outside the household member loop, as it needs to be run only once.  Also reduces the `if` comparisons.

Reply:
Yes, I think that's a good suggestion.